src/Service.Tests/SqlTests/GraphQLMutationTests/DwSqlGraphQLMutationTests.cs

```diff
@@ -0,0 +1,604 @@
+
+            JsonElement response1 = await responseTask1;
+            JsonElement response2 = await responseTask2;
+
+            string responseString1 = response1.ToString();
+            string responseString2 = response2.ToString();
+            string expectedResponse = "{\"result\":\"success\"}";
+            string expectedErrorResponse = "{\"result\":\"item not found\"}";
+
+            // The mutation request that deletes the item is expected to have a valid response
+            // and the other mutation is expected to receive an empty response as it
+            // won't see the item in the table.
+            if (responseString1 == expectedResponse)
+            {
+                Assert.AreEqual(expectedErrorResponse, responseString2);
+            }
+            else if (responseString2 == expectedResponse)
+            {
+                Assert.AreEqual(expectedErrorResponse, responseString1);
+            }
+            else
+            {
+                Assert.Fail("Unexpected failure. Atleast one of the delete mutations should've succeeded");
+            }
+
+        }
+
+        /// <summary>
+        /// Performs concurrent update mutations on the same item and validates that the responses
+        /// returned are consistent
+        /// gQLMutation1 : Updates the title column of Book table to New Title
+        /// gQLMutation2 : Updates the title column of Book table to Updated Title
+        /// </summary>
+        [TestMethod]
+        public override async Task TestParallelUpdateMutations()
+        {
+            string graphQLMutationName = "updatebook";
+            string gQLMutation1 = @"
+                mutation {
+                    updatebook(id : 1, item: { title: ""New Title"" })
+                    {
+                        result
+                    }
+                }";
+
+            string gQLMutation2 = @"
+                mutation {
+                    updatebook(id : 1, item: { title: ""Updated Title"" })
+                    {
+                        result
+                    }
+                }";
+
+            Task<JsonElement> responeTask1 = ExecuteGraphQLRequestAsync(gQLMutation1, graphQLMutationName, isAuthenticated: true);
+            Task<JsonElement> responseTask2 = ExecuteGraphQLRequestAsync(gQLMutation2, graphQLMutationName, isAuthenticated: true);
+
+            JsonElement response1 = await responeTask1;
+            JsonElement response2 = await responseTask2;
+
+            Assert.AreEqual("{\"result\":\"success\"}", response1.ToString());
+            Assert.AreEqual("{\"result\":\"success\"}", response2.ToString());
+        }
+
+        [TestMethod]
+        [Ignore]
+        /// <inheritdoc/>
+        public override Task TestDbPolicyForCreateOperationReferencingFieldAbsentInRequest()
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
add comment for why this is ignored since your changes seem to include DB policies in your query building.

Reply:
fixed.

Reply:
given that you honor policies while generating the corresponding query, why do you think this is not supported for DWsql?